public_html/templates/index.html

```diff
@@ -71,7 +71,7 @@
 					<div class="row-div col-sm-3 text-center wikiproject-div">
 						{% if row.wikiprojects is not empty %}
 							{% for wp in row.wikiprojects %}
-								<span class="row-div wproject">{{ wp }}</span>
+								<a class="row-div wproject" href="#">{{ wp }}</a>
```

Comment:
Yeah, I don't think it's worth separately color-coding each WikiProject. There are about 1000 different WikiProjects so I'm not going to remember what project is what color, and even if I do, it will be roughly the same color as 100 other WikiProjects.